src/sugar.rs

```diff
@@ -112,6 +162,22 @@ macro_rules! proptest {
             proptest_helper!(@_BODY config ($($parm in $strategy),+) $body);
         }
     };
+
+    (|($($arg:tt)+)| $body:block) => {
```

Comment:
You'll want to pull in similar changes to https://github.com/AltSysrq/proptest/commit/c85573f4ed42ea1df350ff33eb1d8737b233e3a8 here